package/test/test.js

```diff
@@ -1,6 +1,20 @@
-const VAPI = require('../src/index.js');
-const VALORANTAPI = new VAPI();
-
-(async () => {
-    console.log(await VALORANTAPI.getCrosshair({code: '0;P;c;5;t;3;o;1;f;0;m;1;0t;4;0l;5;0o;0;0a;1;0f;0;1t;8;1l;3;1o;0;1a;1;1m;0;1f;0', size: 128}));
-})();
+import { spawn } from "child_process";
+import _VAPI from "../dist/index.js";
+const VAPI = new _VAPI();
+const clipboard = (data) => {
+    const proc = spawn("pbcopy");
+    proc.stdin.write(data);
+    proc.stdin.end();
+};
+const res = await VAPI.initUtils();
+// const object = Object.keys(res.data).reduce((acc, cur) => {
+//     if (Array.isArray(res.data[cur])) {
+//         acc[cur] = res.data[cur].slice(0, 2)
+//     } else {
+//         acc[cur] = res.data[cur]
+//     }
+//     return acc
+// }, {})
+// console.log(object)
+// clipboard(JSON.stringify(object))
+//# sourceMappingURL=test.js.map
```

Comment:
This should be deleted or rewritten to a real test.

Reply:
Yeah I'll delete it